src/main/java/com/github/firmwehr/gentle/backend/ir/visit/DjungelskogVisitor.java

```diff
@@ -274,6 +276,28 @@ public String visit(IkeaDiv div) {
 		return result;
 	}
 
+	@Override
+	public String visit(IkeaShr shr) {
+		String result = "";
```

Comment:
git didn't pick that up I guess, but code selection requires each visit method to check if the node has been replace, once you rebase on master. Just a heads up.